### DISCUSSION TITLE
solo q + smurf detection

### DISCUSSION THREAD

**Discussion by quetzalsly** [Heroes-Profile/heroesprofile#1050] [Ideas]
It would be interesting to see what heroes smurfs use as solo q for the highest winrate.
Do we have party size information?
For example mephisto works well with alex globe talent and it inflates the winrate if they are premade.
It would be interesting to see which heroes perform best in each league when playing solo.

**Comment by Zemill** [comment]
You can probably use https://www.heroesprofile.com/Global/Party with the hero filter
